src/content-cli-push.ts

```diff
@@ -233,7 +218,6 @@ class Push {
     }
 }
 
-Push.analysis(commander);
 Push.analysisBookmarks(commander);
```

Comment:
What about bookmarks?

Reply:
I think this is a feature for Studio Analysis, so it should still be working.